### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by tjw0612**
## Why

This pull request is needed because:

1. Added landing page
2. Changed restaurant to chomp-session association

[Link to Card]()

## What

This pull request introduces the following:

 * `Changes introduced`

`Add Implementation details if needed`

### Screenshot

`Attach any screenshots here. Remove when unneeded`

## How to Test

`Give the reviewer(s) some clues on how to test your PR`

## Other

`This section is for things that can't be put into the previous sections. Remove when unneeded`

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+    <h1>Logo</h1><br>
+    <%= simple_form_for :chomp_session, url: root_path, method: :get do |f| %>
+            <div class="search-form-control form-group">
+            <input class="form-control string required" type="text" name="chomped" id="chomped" placeholder="Enter Session Code"/>
+            </div>
+            <button name="button" type="submit" class="btn btn-primary btn-block">
+            Join Existing Group
+            </button>
+            <%= link_to "Create New Group", new_chomp_session_path, class: "btn btn-success btn-block"%><br>
+        <% end %>
```
Comment (mxng) [inline app/views/pages/home.html.erb]: this line does not seem aligned to line 4?

**Review by mxng — APPROVED** [review]
Have one line not aligned only otherwise 👍

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (tjw0612) [reply app/views/pages/home.html.erb]: corrected spacing

**Comment by mxng** [comment]
Think can merge if you want

**Pull request merged**